src/plots/cartesian/axis_defaults.js

```diff
@@ -116,6 +116,9 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
         weight: font.weight,
         style: font.style,
         variant: font.variant,
+        capitalize: font.capitalize,
+        striding: font.striding,
+        shadow: font.shadow,
```

Comment:
Repeating my [comment from the last font PR](https://github.com/plotly/plotly.js/pull/6956#discussion_r1562517865):

> I suspect it would be simpler to use `coerceFont` AND easier to read the logic if it accepted TWO default objects - the first letting you pass on an entire unchanged font object, the second letting you override just one or two parts. If we did that, this call would turn into something like
`Lib.coerceFont(coerce, ‘hoverlabel.font’, globalHoverLabel.font, {color: globalHoverLabel.font.color || hoverBorder})`

In the case here, it would end up like:
```js
Lib.coerceFont(
    coerce,
    'title.font',
    font,
    {size: Lib.bigFont(font.size), color: dfltFontColor}
);
```

Reply:
Thanks very much @alexcjohnson
#6956 was quite big PR and I didn't want to include refactoring changes like that.
But here it looks like the right place to have it.
Please see [150f0c8](https://github.com/plotly/plotly.js/pull/6983/commits/150f0c80295f5faf47b854a83989b7219ff1ac4f)